### PR TITLE
move to PackageLicenseExpression

### PR DESCRIPTION
**Pull request by SimonCropp**
nuget parses and renders this

### CODE REVIEW

File: src/GraphQL/GraphQL.csproj [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <PackageTags>GraphQL;json;api</PackageTags>
     <PackageProjectUrl>https://github.com/graphql-dotnet/graphql-dotnet</PackageProjectUrl>
     <PackageIconUrl>https://raw.githubusercontent.com/graphql-dotnet/graphql-dotnet/master/assets/logo.64x64.png</PackageIconUrl>
```
Comment (sungam3r) [inline src/GraphQL/GraphQL.csproj]: Could you please also change this to <PackageIcon> ?
```c#
<PackageIcon>logo.64x64.png</PackageIcon>
  <ItemGroup>
    <None Include="..\..\assets\logo.64x64.png" Pack="true" PackagePath="" />
  </ItemGroup>
```

Comment (SimonCropp) [reply src/GraphQL/GraphQL.csproj]: wont that conflict with https://github.com/graphql-dotnet/graphql-dotnet/pull/1478  ?

Comment (sungam3r) [reply src/GraphQL/GraphQL.csproj]: I did not see, just answered in order to all incoming mail.

Comment (SimonCropp) [reply src/GraphQL/GraphQL.csproj]: i rebased of master. your comment should be addressed now

**Pull request merged**